tests/test_samples.py

```diff
@@ -1347,6 +1347,8 @@ def test_old_gui():
     with pytest.raises(NotImplementedError):
         make_1d_axes(['x0', 'y0'], tex={'x0': '$x_0$', 'y0': '$y_0$'})
 
+    with pytest.raises(NotImplementedError):
+        samples.dlogX(1000)
```

Comment:
A bit funny, that the removal of `dlogX` is what introduces its first test...

Should we add a test for `logdX` to at least cover its existence in the tests?